pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -47,12 +47,8 @@ const (
 	baseOsObj = "baseOs.obj"
 	agentName = "verifier"
 
-	persistDir            = "/persist"
-	objectDownloadDirname = persistDir + "/downloads"
-
-	rootCertDirname    = "/config"
-	rootCertFileName   = rootCertDirname + "/root-certificate.pem"
-	certificateDirname = persistDir + "/certs"
+	rootCertDirname  = "/config"
```

Comment:
I would think this should be the global identityDirname you put in the shared file and nothing else.

Reply:
Done.. Removed it..